demos/complementary-color-prediction/complementary-color-prediction.ts

```diff
@@ -0,0 +1,355 @@
+/* Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Reminder to run `npm run lint` to check for lint errors. Thank for your patience before we automate this with travis CI.